core/modules/main/templates/_issue_workflow_transition.inc.php

```diff
@@ -2,7 +2,7 @@
     <?php if ($issue->isWorkflowTransitionsAvailable()): ?>
         <?php foreach ($issue->getAvailableWorkflowTransitions() as $transition): ?>
             <?php if ($transition instanceof \thebuggenie\core\entities\WorkflowTransition && $transition->hasTemplate()): ?>
-                <?php include_component($transition->getTemplate(), compact('issue', 'transition', 'ajax')); ?>
+                <?php include_component($transition->getTemplate(), compact('issue', 'transition', isset($ajax)?'ajax':array())); ?>
```

Comment:
it's preferrable to initialize with empty values in the associated `componentX()` method, so you don't have to do these checks everywhere. 
```
if (!isset($this->ajax)) $this->ajax = null;
```
in the component method should do it.